core/src/main/scala/org/apache/spark/sql/delta/commands/CloneTableBase.scala

```diff
@@ -278,8 +278,11 @@ abstract class CloneTableBase(
           val copiedFile = fileToCopy.copy(dataChange = true)
           opName match {
             case CloneTableCommand.OP_NAME =>
+              // CLONE does not preserve Row IDs and Commit Versions
```

Comment:
why not?

Reply:
It is explained in the design document. The fundamental problem is that there is no way to remove a file and add it back with a different `baseRowId` in the same commit. This causes issues with incremental clones.